app/services/totp_deleter.go

```diff
@@ -0,0 +1,20 @@
+package services
+
+import (
+	"github.com/keratin/authn-server/app/data"
+	"github.com/pkg/errors"
+)
+
+//TOTPDeleter removes TOTP from the specified account
+func TOTPDeleter(accountStore data.AccountStore, accountID int) error {
+	//Delete totp secret in database
+	affected, err := accountStore.DeleteTOTPSecret(accountID)
+	if err != nil {
+		return errors.Wrap(err, "TOTPDeleter")
+	}
+	if !affected {
+		return errors.New("unable to delete totp secret")
+	}
```

Comment:
should we handle this gracefully? i can't think of downsides to returning nil in this case.